spec/fixtures/files/prometheus1.sysv

```diff
@@ -51,12 +51,12 @@ start() {
         [ -f $PID_FILE ] && rm $PID_FILE
         daemon --user=prometheus \
             --pidfile="$PID_FILE" \
-            "$DAEMON" -log.format logger:stdout -config.file=/etc/prometheus/prometheus.yaml \
+            "'$DAEMON' -log.format logger:stdout -config.file=/etc/prometheus/prometheus.yaml \
             -web.console.templates=/usr/local/share/prometheus/consoles \
             -web.console.libraries=/usr/local/share/prometheus/console_libraries \
             -storage.local.path=/var/lib/prometheus \
             -storage.local.retention=360h \
-             >> "$LOG_FILE" &
+            >> '$LOG_FILE' 2>&1 &"
```

Comment:
can you explain what this actually fixes? support paths with whitespace?

Reply:
Hi, without the additional double quotes parent startup processes remain after startup and logfiles are touched as root.
e.g.

`
/bin/bash /etc/init.d/prometheus restart (root)
\\_ runuser -s /bin/bash prometheus -c ulimit -S -c 0 >/dev/null ... (root)

  \\_ bash -c ulimit -S -c 0 >/dev/null 2>&1 ; /usr/local/bin/prometheus --config.file... (prometheus)

   \\_ /usr/local/bin/prometheus --config.file=/etc/prometheus/prometheus.yaml --web.console.... (prometheus)
`

compared to

`493      56754  6.0  0.0 774908 53652 ?        Sl   09:18   0:00 /usr/local/bin/prometheus --config.file=/etc/prometheus/prometheus.yaml --web.console.templates=/usr/local/share/prometheus/consoles --web.console.libraries=/usr/local/share/prometheus/console_libraries --st
`